lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,17 +552,25 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.warning(`The cloud instance discovery network request's status code is: ${response.status}`);
+                this.logger.warning("metadata is an empty list.");
+                
+                const tenantDiscoveryEnpoint = response.body.tenant_discovery_endpoint;
+                const tenantDiscoveryEnpointLoggerHelper = tenantDiscoveryEnpoint.length ? `: ${tenantDiscoveryEnpoint.length}` : " an empty string";
+                this.logger.warning(`tenant_discovery_endpoint is${tenantDiscoveryEnpointLoggerHelper}`);
```

Comment:
The pattern is to log 2 strings: 

- the actual tenant id as pii log statement
- a placeholder as non-pii log statement